WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -300,6 +300,10 @@ private extension ImageAndTitleAndTextTableViewCell {
     func configureTitleAndTextStackView() {
         titleAndTextStackView.spacing = 2
     }
+    func configureBackground() {
```

Comment:
Super nit: We want to be mindful of the space between code to make it readable, so an extra space between functions is usually preferred.

Reply:
Sure thing!